src/kudu/consensus/metadata.proto

```diff
@@ -101,7 +101,13 @@ message RaftPeerPB {
   // manually specifying a configuration for Master/CatalogManager);
   // permanent uuid can be retrieved at a later time through RPC.
   optional bytes permanent_uuid = 1;
+
+  // TODO(ritwikyadav): Deprecate member_type once migration to
+  // `election_member_type` and `data_member_type` is complete.
   optional MemberType member_type = 2;
+  optional MemberType election_member_type = 7;
+  optional MemberType data_member_type = 8;
```

Comment:
I'm confused about the purpose of this. MemberType is defined in consensus/metadata.proto as follows:
```
enum MemberType {
    UNKNOWN_MEMBER_TYPE = 999;
    NON_VOTER = 0;
    VOTER = 1;
  };
```
Can we meet to discuss the proposed design, or write it down to review it?